Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -0,0 +1,282 @@
+    user_data = client.get_user(user, '*')
+    user_readable, user_outputs = parse_outputs(user_data)
+    human_readable = tableToMarkdown(name=f"{user} was created successfully:", t=user_readable, removeNull=True)
+    outputs = {'MSGraphUser(val.ID == obj.ID)': user_outputs}
+    return human_readable, outputs, user_data
+
+
+def update_user_command(client: MsGraphClient, args: Dict):
+    user = args.get('user')
+    updated_fields = args.get('updated_fields')
+
+    client.update_user(user, updated_fields)
+    return get_user_command(client, args)
+
+
+def get_delta_command(client: MsGraphClient, args: Dict):
+    properties = args.get('properties', '') + ',userPrincipalName'
+    users_data = client.get_delta(properties)
+    headers = list(set([camel_case_to_readable(p) for p in argToList(properties)] + ['ID', 'User Principal Name']))
+
+    users_readable, users_outputs = parse_outputs(users_data)
+    human_readable = tableToMarkdown(name='All Graph Users', headers=headers, t=users_readable, removeNull=True)
+    outputs = {'MSGraphUser(val.ID == obj.ID)': users_outputs}
+    return human_readable, outputs, users_data
+
+
+def get_user_command(client: MsGraphClient, args: Dict):
+    user = args.get('user')
+    properties = args.get('properties', '*')
+    user_data = client.get_user(user, properties)
+
+    user_readable, user_outputs = parse_outputs(user_data)
+    human_readable = tableToMarkdown(name=f"{user} data", t=user_readable, removeNull=True)
+    outputs = {'MSGraphUser(val.ID == obj.ID)': user_outputs}
+    return human_readable, outputs, user_data
+
+
+def list_users_command(client: MsGraphClient, args: Dict):
+    properties = args.get('properties', 'id,displayName,jobTitle,mobilePhone,mail')
+    next_page = args.get('next_page', None)
+    users_data, result_next_page = client.list_users(properties, next_page)
+    users_readable, users_outputs = parse_outputs(users_data)
+    metadata = None
+    outputs = {'MSGraphUser(val.ID == obj.ID)': users_outputs}
+
+    if result_next_page:
+        metadata = "To get further results, enter this to the next_page parameter:\n" + str(result_next_page)
+
+        # .NextPage.indexOf(\'http\')>=0 : will make sure the NextPage token will always be updated because it's a url
+        outputs['MSGraphUser(val.NextPage.indexOf(\'http\')>=0)'] = {'NextPage': result_next_page}
+
+    human_readable = tableToMarkdown(name='All Graph Users', t=users_readable, removeNull=True, metadata=metadata)
+
+    return human_readable, outputs, users_data
+
+
+def main():
+    params: dict = demisto.params()
+    url = params.get('host', '').rstrip('/') + '/v1.0/'
+    tenant = params.get('tenant_id')
+    auth_and_token_url = params.get('auth_id', '')
+    enc_key = params.get('enc_key')
+    verify = not params.get('insecure', False)
+    self_deployed: bool = params.get('self_deployed', False)
+    # TODO: check it does the same as handle proxy
```

Comment:
what about this TODO ?

Reply:
I checked it and forgot to remove this line. sorry.